secretstores/secret_store.go

```diff
@@ -11,4 +11,6 @@ type SecretStore interface {
 	Init(metadata Metadata) error
 	// GetSecret retrieves a secret using a key and returns a map of decrypted string/string values
 	GetSecret(req GetSecretRequest) (GetSecretResponse, error)
```

Comment:
@yaron2 Do we still need single secret get in secret component? it seems like it is redundant.

Reply:
We have in Dapr a Single Get API and a Bulk Get API. We can change the implementation to use the Bulk and return the first item, I guess.

Reply:
`GetSecret` performance will always be better than `GetSecrets` when retrieving a single secret cause `GetSecrets` usually requires on call to get all the keys and then one call for each found key.

This becomes relevant specially when using `secretKeyRef` in the components. which should load as fast as possible, while the use case for  `GetSecrets`  is bulk loading from a dedicated store when loading your app or lazy loading when needed.

Reply:
That's a good point.

Reply:
ok make sense.

Then please keep naming convention consistent with the other bulk api. In the case of store, we use `Bulk` prefix. It would be easier to recognize suffix `s`. 
https://github.com/dapr/components-contrib/blob/master/state/store.go#L15

Reply:
Changed the API to use the Bulk prefix...